src/app/app.ts

```diff
@@ -1223,12 +1228,12 @@ function waterclick(): void {
     document.getElementById("sleep-img").style.color = '#999393';
     document.getElementById("water-img").style.color = '#FFFFFF';
     document.getElementById("food-img").style.color = '#999393';
-    document.getElementById("title").innerHTML = 'Water Consumption <span id="watertitle-annotation1">Daily Average</span><span id="watertitle-annotation2">Target</span>';
-    document.getElementById("subtitle").innerHTML = '<span id="watersubtitle-annotation1">4.32 litres</span><span id="watersubtitle-annotation2">7 litres</span>'
-    document.getElementById("watersubtitle-annotation1").style.color = '#3B61E9';
-    document.getElementById("watersubtitle-annotation2").style.color = '#3B61E9';
+    document.getElementById('chart-header-title').innerHTML = '<p class="steps-ch-title" id="title"> Water Consumption </p>'
+    document.getElementById('total-value').innerHTML = '   <p class="water-total-title" id="title"> Target </p> <p class="water-st-today subtitle" id="chart-water-total-subtitle"> 7 litres </p>'
+    document.getElementById("average-value").innerHTML = '<p class="water-daily-avg-title" id="title"> Daily Average </p> <p class="water-st-daily-average subtitle" id="chart-water-average-subtitle"> 4.32 litres </p>'
+    document.getElementById('chart-water-total-subtitle').style.color = '#3B61E9'
```

Comment:
Element is added only in the previous line, directly add the style there.
Also, update the class, id names to simpler ones and avoid generic names

Reply:
Style has applied inline to the element. 
Generic class names has been avoided instead simpler names are used